src/components/fabSpeedDial/fabSpeedDial.scss

```diff
@@ -109,6 +107,15 @@ md-fab-speed-dial {
   }
 
   /*
+   * Hide some graphics glitches if switching animation types
+   */
+  &.md-fling-remove, &.md-scale-remove {
+    .md-fab-action-item > * {
+      visibility: hidden;
```

Comment:
The speed dial currently uses flex to order/align both the trigger and the actions. When I used `display: none` with the speed dial in direction `up`, the actions height changed and the trigger jumped into the incorrect position :/

I think ideally we'll rewrite this in material2 to use absolute positioning that is calculated in the JS so we don't have to worry about issues like this.